setup.cfg

```diff
@@ -0,0 +1,43 @@
+[metadata]
+name = pytsql
+version = attr: pytsql.__version__
+description = `Pytsql` allows to run mssql sripts, typically run via GUIs, via CLI.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/quantco/pytsql
+author = Alex Gonopolskiy, Kevin Klein, Encho Mishinev
+author_email = agonopol@gmail.com, kevin.klein@quantco.com, encho.mishinev@quantco.com
+maintainer = Kevin Klein
+maintainer_email = kevin.klein@quantco.com
+keywords =
+    sql
+    mssql
+    databases
+    automation
+    parametrization
+classifiers =
+    Development Status :: 5 - Production/Stable
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Database
+
+[options]
+include_package_data = true
+install_requires =
+    sqlalchemy >=1.4
+    antlr4-python3-runtime ==4.9.2
+    pyodbc >=4.0.30
+python_requires = >=3.7
+package_dir=
+    =src
```

Comment:
Can this be on the previous line?

Reply:
It can. However, this parameter is a dictionary, which can span over multiple lines, so maybe leaving it the way it is would be more indicative of that? 

For some reason this is the way the example has it in [setuptools src layout].(https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#using-a-src-layout).